### ISSUE TITLE
Files App crashing on right click

### ISSUE THREAD

**Issue by Sunrise-Sea** [files-community/Files#9643]
### Right clicking a file crashes the app

Hi, 

I had recently installed the Files App and was going to use it for, I think the second time, but when I _right clicked_ a file while 
browsing, it just crashed and then closed. A notification popped up informing an error.

Please look into this issue. Or feel free to close it if it was a duplicate one.

### Steps To Reproduce

1. **It's simple.**
2. Right click any file in whichever location **except the disk drives** and **the things pinned in the details pane**
3. The whole app crashes.

### Expected behavior

I expected the context menu to show up when I right clicked, it notified me of an error that the developers weren't prepared for.

### Files Version

Version: 2.3.3.0 OS Version: 10.0.22000.832

### Windows Version

Windows 11 21H2 22000.832

### Relevant Assets/Logs

_Here are the log files._

[debug.log](https://github.com/files-community/Files/files/9247749/debug.log)
[debug_fulltrust.log](https://github.com/files-community/Files/files/9247750/debug_fulltrust.log)

**Comment by gave92** [comment]
Turn off the File Tags feature in settings/experimental.

**Comment by JJSax** [comment]
I'm having a similar issue.  For me, I right click on several files without issue.  Transferring .zip files & right clicking .zip files causes the app to crash.  I do not have experimental tags enabled.  I did not get a popup error.  I just simply crashed.  I'm unsure if it's the same bug or not.

Files:
Version: 2.3.6.0
OS Version: 10.0.19043.1889
[debug.log](https://github.com/files-community/Files/files/9339734/debug.log)
[debug_fulltrust.log](https://github.com/files-community/Files/files/9339735/debug_fulltrust.log)
